avocado/plugins/human.py

```diff
@@ -39,7 +38,7 @@ class Human(ResultEvents):
                       'CANCEL': output.TERM_SUPPORT.CANCEL}
 
     def __init__(self, args):
-        self.log = logging.getLogger("avocado.app")
+        self.log = LOG_UI
```

Comment:
@ldoktor my personal opinion is that "probably" deeper changes will be needed here when multi-job code is exercised.  I wouldn't keep the "probable" future as the reason for this *actual* code.

But, in the end, your version of the code also works, and it can be seen as your style, for these files (even if it differs from others).